client/src/utils/Store.js

```diff
@@ -11,19 +11,24 @@ import {
 axios.defaults.withCredentials = true;
 export default class API {
   constructor() {
-    this.plot = '';
-    this.models = new Map();
+    this.state = {
+      plot: '',
+      models: {},
+    };
   }
 
   /**
    * initializes cache
    */
   init = async () => {
     const k = await keys();
+    const models = new Map();
     k.forEach((key) => {
-      this.models.set(key, get(key));
+      const obj = get(key);
+      models.set(key, obj);
+      this.state.models[key] = obj;
```

Comment:
Does this work? Object is a promise. Dont you have to convert models after promise.all?

Reply:
This isnt working, you need to convert the map into an object after the await Promise.all(). 
<img width="476" alt="Screen Shot 2020-11-29 at 3 13 05 PM" src="https://user-images.githubusercontent.com/13951813/100535615-74e97d00-3255-11eb-8c70-d9c26bfe5a1d.png">